pkg/collector/corechecks/containers/containerd.go

```diff
@@ -40,13 +41,14 @@ type ContainerdCheck struct {
 	core.CheckBase
 	instance *ContainerdConfig
 	sub      *subscriber
+	filters  *containers.Filter
 }
 
 // ContainerdConfig contains the custom options and configurations set by the user.
 type ContainerdConfig struct {
-	Tags          []string `yaml:"tags"`
-	Filters       []string `yaml:"filters"`
-	CollectEvents bool     `yaml:"collect_events"`
+	Tags              []string `yaml:"tags"`
+	containerdFilters []string `yaml:"filters"`
```

Comment:
IIRC only public fields are marshalled/unmarshalled from yaml, you might want to keep `ContainerdFilters`